ehitchhiking-ui/package.json

```diff
@@ -11,14 +11,17 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.1.0",
+    "@angular/animations": "^8.1.0",
```

Comment:
Why do you use ^? Are you ready to the upgrade on 9.x.x. version?

Reply:
fixed with ~